ERP-BACKEND/app/domains/finance/interface.py

```
@@ -0,0 +1,105 @@
+class PaymentCreate(BaseModel):
+    invoice_id: int
+    amount: float
+    payment_method: str
+    payment_date: date
+
+
+def build_router() -> APIRouter:
+    router = APIRouter()
+
+    @router.post("/invoices")
+    def create_invoice(data: InvoiceCreate, db: Session = Depends(get_db),
```

Ruff (0.16.1)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)